libs/sdk/e2e-schematics/src/generators/configure-percy/index.spec.ts

```diff
@@ -1,71 +1,48 @@
 import { cypressProjectGenerator } from '@nrwl/cypress';
-import { createTreeWithEmptyV1Workspace } from '@nrwl/devkit/testing';
-
-import { updateJson } from '../../utils';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
 
 import configurePercy from './index';
 
 describe('configure-percy', () => {
   it('should import percy', async () => {
-    const tree = createTreeWithEmptyV1Workspace();
+    const tree = createTreeWithEmptyWorkspace();
     await cypressProjectGenerator(tree, {
       name: `cypress`,
       baseUrl: 'https://example.com',
     });
-    expect(tree.exists('apps/cypress/src/support/index.ts')).toBeTruthy();
+    expect(tree.exists('apps/cypress/src/support/e2e.ts')).toBeTruthy();
     expect(
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      tree.read('apps/cypress/src/support/index.ts')!.toString()
+      tree.read('apps/cypress/src/support/e2e.ts')!.toString()
     ).not.toContain('percy');
     await configurePercy(tree, { name: 'cypress' });
     expect(
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      tree.read('apps/cypress/src/support/index.ts')!.toString()
+      tree.read('apps/cypress/src/support/e2e.ts')!.toString()
     ).toContain('percy');
   });
 
-  it('should not make changes without cypress.json', async () => {
-    const tree = createTreeWithEmptyV1Workspace();
-    await cypressProjectGenerator(tree, {
-      name: `cypress`,
-      baseUrl: 'https://example.com',
-    });
-    tree.delete(`apps/cypress/cypress.json`);
-    const beforeLength = tree.listChanges().length;
-    await configurePercy(tree, { name: 'cypress' });
-    expect(tree.listChanges().length).toEqual(beforeLength);
-  });
-
-  it('should handle missing supportFile config', async () => {
-    const tree = createTreeWithEmptyV1Workspace();
+  it('should generate cypress.config.ts', async () => {
```

Comment:
Cypress 10 uses `cypress.config.ts` rather than `cypress.json`. We use the defaults except we turn off video, and that is now handled using a template file rather than updating a JSON file.